kubernetes/kubapply

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+if [ ! -f "Vars" ]; then
+   echo "File Vars does not exist. Create A file Vars named Vars containing the variables."
+   exit
+fi
+
+FILE=${1}
+
+if [ ! $FILE ]; then
+   echo "Please Provide a file"
+   exit
+fi
+
+
+if [ ! -f $FILE ]; then
+   echo "File $FILE does not exist"
+   exit
+fi
+
+
+COM="cat $FILE | sed \""
+
+for var in $(cat Vars); do
+   COM="$COM;s/${var/=//}/g"
+done
+
+COM="$COM\" | kubectl apply -f -"
+
+
+eval "$COM"
```

Comment:
I am frankly not happy about the fact that even after making so many pull requests you have not added a blankline at the end! Please follow the reviews we make, they are really important to keep the project up and running. :cry:

Reply:
Sorry about that